Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM debian:stable
```

Comment:
I guess it always works if this is compiled on Debian? If it works it works.

Reply:
Sort of. The issue is really that the host's glibc version is likely to be too recent. If that's the case, it won't run on Debian 11.